### PR TITLE
Add device monitor message.

### PR DESCRIPTION
**Pull request by wltr**
This message adds support for internal voltage and die temperature measurements on Zynq based devices.

@mfine @denniszollo @switanis

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/piksi.yaml [hunk 1]
```diff
@@ -282,3 +282,28 @@ definitions:
...
+            type: float
+            units: V
+            desc: Processor VCCINT
+        - cpu_vccaux:
+            type: float
+            units: V
+            desc: Processor VCCAUX
+        - cpu_temperature:
+            type: float
+            units: C
```
Comment (ljbade) [inline spec/yaml/swiftnav/sbp/piksi.yaml]: degrees C

File: spec/yaml/swiftnav/sbp/piksi.yaml [hunk 2]
```diff
@@ -282,3 +282,28 @@ definitions:
...
+            desc: Processor temperature
+        - fe_temperature:
+            type: float
+            units: C
```
Comment (ljbade) [inline spec/yaml/swiftnav/sbp/piksi.yaml]: degrees C

**Comment by ljbade** [comment]
Nice

**Comment by JoshuaGross** [comment]
Looks good, I would also like to see more specific units like Leith indicated. Also, run the code generators if you can.

**Comment by switanis** [comment]
1. I strongly recommend to use scaled integers for passing values instead of floats. Integers can be shorter, are easier to deal and examine, clearly define range and resolution, are not subject of rounding, faster to manipulate. For voltage we can use u16 scaled by 1000. That will give range 0 to 65.535 V with 1mV resolution. Or make it s16 to cover from -32.768 to 32.767 V with 1mV res. For temp we can use s16 scaled by 100. That will give us range -327.68 (really -273.15) to 327.67 deg C with 0.01 deg C resolution. 

2. Can we measure board supply voltage as well? That'd be the most useful for customers.

**Comment by wltr** [comment]
Thanks for the comments! 

**Updates:**
- Use `degree C`
- Use `s16` with scaling factor
- Add `dev_vin` to monitor VIN on the GNSS module (has yet to be implemented in firmware)

@ljbade @JoshuaGross @switanis

**Comment by JoshuaGross** [comment]
Looks good. Going to fix the merge conflict and pull in manually.

**Pull request closed without merging**